src/redecomp/RedecompMesh.cpp

```diff
@@ -296,7 +296,7 @@ void RedecompMesh::BuildRedecomp()
   }
 
   // Finalize mesh topology
-  FinalizeTopology();
+  FinalizeTopology(false);
```

Comment:
Maybe indicate what passing 'false' does here. Or initialize some boolean named something to make this clearer. Is there a case where you'd pass 'true'?

Reply:
Ahh this is to prevent boundary data (elements, attributes) from being created, since 1) it isn't clear what we would even want to be considered the boundary mesh and 2) the boundary mesh isn't used anywhere in Tribol.  Also, it sets the boundary attributes to zero which causes a lot of warnings in the output.  I'll name the variable.